eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -72,6 +73,17 @@ void checksPacks(final String pack) throws IOException {
         );
     }
 
+    @ParameterizedTest
+    @ClasspathSource(value = "org/eolang/maven/xmir2xmir/", glob = "**.yaml")
+    @ExtendWith(XcopCondition.class)
+    void checksXmirToXmir(final String pack, @TempDir final Path dir) {
```

Comment:
@levBagryansky maybe let's call it something like `checksXmirToXmirPacks` or `checksXmirToXmirPrePacks`

Reply:
@maxonfjvipon done